packages/teleport-component-generator/src/resolver/utils.ts

```diff
@@ -127,6 +132,20 @@ export const resolveElement = (element: UIDLElement, options: GeneratorOptions)
     originalElement.attrs = resolveAttributes(mappedElement.attrs, originalElement.attrs)
   }
 
+  if (originalElement.attrs && attributesMapping) {
+    const attrsKeys = Object.keys(originalElement.attrs)
+
+    attrsKeys
+      .filter((key) => attributesMapping[key])
+      .forEach((key) => {
+        originalElement.attrs = {
+          ...originalElement.attrs,
```

Comment:
you don't need to spread the originalElement.attrs here, only set the new value:
`originalElement.attrs[attributesMapping[key]] = originalElement.attrs[key]`